src/components/ParticipantList.jsx

```diff
@@ -54,11 +59,51 @@ export default function ParticipantList({ studyID, title, onParticipantSelect, s
         })()
     }, [participants])
 
+
     let addParticipant = async () => {
         let newCount = state.newCount
         let ids = []
-        for (let i = 0; i < newCount; i ++)
-            ids = [...ids, (await LAMP.Participant.create(studyID, { study_code: '001' })).data]
+        let id = []
+        let tempPassword = []
```

Comment:
You should be able to delete these variables here and just declare them inside the loop as 
```
let id = ...
let tempPassword = ...
```

Also, you've declared them as empty arrays, but you later assign them as strings -- avoid changing variable types.

Reply:
change made